addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1891,8 +1892,14 @@ var SnippetsMenu = Widget.extend({
             this.invisibleDOMMap = new Map();
             const $invisibleDOMPanelEl = $(this.invisibleDOMPanelEl);
             $invisibleDOMPanelEl.find('.o_we_invisible_entry').remove();
-            const $invisibleSnippets = globalSelector.all().find('.o_snippet_invisible').addBack('.o_snippet_invisible');
+            const $selector = this.options.enableTranslation ? $('#wrapwrap') : globalSelector.all();
```

Comment:
Not a fan of mentioning wrapwrap directly like that, don't we have a function / variable holding the editable area somewhere that you could use here? If not, that's ok we'll leave that to a refactoring in master.

Reply:
Unfortunately, using `getEditableArea()` will not work as it is searching for `.o_editable` element so, in translate mode, `getEditableArea()` will return all the translatable text element but will not return the `header`, the `wrap` element, the `footer` etc... as in non translate mode.

Reply:
I was not thinking about `getEditableArea` but rather something like `this.theElOnWhichTheEditorWasInstantiated` or even `document.body` :wink: But let's leave it like that for now to be sure, this will be refactored at some point :+1: